### PR TITLE
Minor FlxDrawTrianglesItem optimization

### PR DESCRIPTION
**Pull request by MaybeMaru**
Uses openfl's vector length function instead of splice for a minor performance boost clearing the vectors.

### CODE REVIEW

File: flixel/graphics/tile/FlxDrawTrianglesItem.hx [hunk 1]
```diff
@@ -101,10 +101,17 @@ class FlxDrawTrianglesItem extends FlxDrawBaseItem<FlxDrawTrianglesItem>
 	override public function reset():Void
 	{
 		super.reset();
+		#if (flash || openfl >= "4.0.0")
```
Comment (Geokureli) [inline flixel/graphics/tile/FlxDrawTrianglesItem.hx:104]: we don't support openfl versions older than 9.3.2, we plan to remove the other checks like this, soon. So let's not add more

Comment (MaybeMaru) [reply flixel/graphics/tile/FlxDrawTrianglesItem.hx:104]: should i remove the check then?

Comment (Geokureli) [reply flixel/graphics/tile/FlxDrawTrianglesItem.hx:104]: yes, please

Comment (MaybeMaru) [reply flixel/graphics/tile/FlxDrawTrianglesItem.hx]: there we go

**Comment by Geokureli** [comment]
Thanks!

**Pull request merged**